buildstockbatch/gcp/gcp.py

```diff
@@ -942,8 +970,8 @@ def start_combine_results_job_on_cloud(self, results_dir, do_timeseries=True):
                             image=self.repository_uri + ":" + self.job_identifier,
                             resources=run_v2.ResourceRequirements(
                                 limits={
-                                    "memory": f"{pp_env_cfg.get('memory_mib', 4096)}Mi",
-                                    "cpu": str(pp_env_cfg.get("cpus", 2)),
+                                    "memory": f"{pp_env_cfg.get('memory_mib', self.DEFAULT_PP_MEMORY_MIB)}Mi",
```

Comment:
should these use `GcpBatch` rather than `self` (or why is it inconsistent with the other usage above)?